.gitattributes

```diff
@@ -1,4 +1,4 @@
-* text eol=lf
+* text=auto
```

Comment:
I think we should limit `text=auto` to only `.exe` files and keep `eol=lf` for all other files. Setting to auto could break our prettier config.

Reply:
Okey, I think in this case just mark *.exe as a binary (alias for `-text -diff`) is the solution. I've already tested it and it's work.